Source/WebKit/_WebKit_SwiftUI/Implementation/WebViewRepresentable.swift

```diff
@@ -37,6 +38,7 @@ struct WebViewRepresentable {
 
         let parent = CocoaWebViewAdapter()
         parent.webView = page.backingWebView
+        parent.geometrySafeAreaInsets = geometrySafeAreaInsets
```

Comment:
Do we need to set this in `makePlatformView` at all? Because I recall it being zero at the beginning and then updated later in `updatePlatformView` anyway.

Reply:
After talking to @rr-codes, we can leave this in, just to be safe.